### ISSUE TITLE
Host

### ISSUE THREAD

**Issue by BlueWebb** [LeaPhant/flowabot#27]
Hello, I would like to put the bot on **hosting**
I am an ordinary user, installed the bot according to the instructions
And now I would like the bot to work on the host, I tried to put it on the host "Heroku" but I did not succeed :(
I looked at different manuals, but they did not help me.
**I just wanted to get some advice from you**, and maybe someone else can help with that, I'm sure it's not difficult to do (probably)

(I don't know English well, so I used Google Translate xD)

**Comment by LeaPhant** [comment]
Okay, I can try to help you but I have no experience with Heroku myself.

**Comment by BlueWebb** [comment]
Thank you, this hosting is free.
I can start the bot through the Heroku console on the site (heroku run bash)
![2](https://user-images.githubusercontent.com/52789281/62430361-9b141100-b724-11e9-9a27-d133d5f50fab.png)
![3](https://user-images.githubusercontent.com/52789281/62430360-9a7b7a80-b724-11e9-90a4-373d5f71dab7.png)

but after closing the console, the bot is disabled and the changes cannot be saved.

When I launch the bot through the "worker node index.js", I get an error - "Error: module" ./config.json "could not be found."
![4](https://user-images.githubusercontent.com/52789281/62430413-24c3de80-b725-11e9-87e7-cb9ab89af75d.png)
I see that the bot can work fine if I run it through the console, but I can not understand how to make the bot run through the "worker node index.js."

I understand that when I run the bot on the server, it should immediately know my token, client id and other information, but I do not understand how to give him this information before the launch, or after the launch. Maybe that's why it doesn't work properly.

I have little knowledge of this, so I may not know something, and make stupid mistakes :)

**Comment by LeaPhant** [comment]
It seems like the worker runs in a different directory where no config.json exists, and as you can't access console input it won't be able to run the configuration wizard. Maybe you can find a way to copy the config.json into the worker's directory.

**Comment by BlueWebb** [comment]
I managed to launch a bot on the Heroku host!

I launched the bot on my PC via **Ubuntu**, then I ran `npm run config` command, set all the settings, then I looked at the files in the directory using the `ls -la` command and saw the file `config.json` there, I opened it through command `cat config.json`, copied all the contents, and created the same file on github.
Done.
It is important that your repository is **private** :)

Thank you for paying attention to my problem :)
